deploy/snapshot/internal/pagebroker/wire_test.go

```
@@ -0,0 +1,112 @@
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+func TestCppWireCompatibility(t *testing.T) {
+	root := filepath.Join("..", "..", "pagebroker")
+	t.Cleanup(func() {
+		if out, err := exec.Command("make", "-C", root, "clean").CombinedOutput(); err != nil {
+			t.Errorf("clean wire fixture: %v\n%s", err, out)
+		}
+	})
+	if out, err := exec.Command("make", "-C", root, "wire-fixture").CombinedOutput(); err != nil {
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)